templates/README.md.ts

```diff
@@ -137,11 +133,11 @@ The template provides you with basic tests for the adapter startup and package f
 It is recommended that you add your own tests into the mix.
 
 ` : ""}### Publishing the ${isAdapter ? "adapter" : "widget"}
-${useGithubActions ? `Since you have chosen GitHub Actions as your CI service, you can 
+Since you have chosen GitHub Actions as your CI service, you can 
```

Comment:
The developer didn't choose anything ;-). I would suggest to simplify this sentence.

Reply:
reworded, please check